kymatio/scattering2d/frontend/numpy_frontend.py

```diff
@@ -35,7 +35,7 @@ def scattering(self, input):
 
         input = input.reshape((-1,) + signal_shape)
 
-        S = scattering2d(input, self.pad, self.unpad, self.backend, self.J,
+        S = scattering2d(input, self.pad, self.unpad, self.pre_pad, self.backend, self.J,
```

Comment:
The `self.pre_pad` doesn't seem to be used.

Reply:
Removed.